### ISSUE TITLE
How can I enable this work with thousands of services

### ISSUE THREAD

**Issue by gyliu513** [rshriram/istio_federation_demo#5]
@rshriram thanks for the tutorial, it woks well in my local Kubernetes Cluster.

But here I have one question want to get your help, I found that we are now mainly configuring virtualservice,  destinationrules, etc as proxy. But the problem is that, in a large scale cluster, if I have thousands of services, I will need to configure thousands of virtualservice,  destinationrules to support the communication between different clusters, it is a bit complex, comments?

**Comment by rshriram** [comment]
It certainly is. But the rationale is that people don't expose two clusters with 1000 services each, directly. They pick a subset of services in each cluster to expose to other clusters. If we really want a 1000x1000, then we need a higher level tooling to automatically generate the required configuration artifacts and push to istio.

**Comment by gyliu513** [comment]
Thanks  @rshriram , so another issue is for tracing, seems with this solution, there is no central UI which can enable me to view all of the mesh cross different clouds/clusters as I need to deploy Jaeger or Kiali in its own cloud/cluster?

**Comment by rshriram** [comment]
Yes. The idea is that these are typically provided as part of the cloud

**Comment by gyliu513** [comment]
Then the problem is that I will not able to view the connected services in remote cluster from my local cluster, which is not convenient for debug and tracing?
